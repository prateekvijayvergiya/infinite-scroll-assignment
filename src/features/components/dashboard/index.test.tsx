import Component from './'
import makeSnapshot from './../../testing/makeSnapshot'

describe('Dashboard', () => {
  it('renders correctly', () => {
    makeSnapshot(<Component />)
  })
})
