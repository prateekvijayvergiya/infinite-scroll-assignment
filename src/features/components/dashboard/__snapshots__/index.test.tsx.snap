// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
    >
      Welcome to Marriage portal
    </h4>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <img
      alt="marriage-logo"
      height={200}
      src="marriage.jpeg"
      width={200}
    />
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Go to Login page
    </span>
  </button>
</div>
`;
